src/Household.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "Coordinates/Location.hpp"
+
+struct Household
+{
+	int FirstPerson;
+	unsigned short int nh; // number people in household
+	Location loc;
+	unsigned short int nhr;
+
+	float distance_to(Household other);
+	float distance_squared_to(Household other);
```

Comment:
```suggestion
	float distance_to_squared(Household other);
```

Reply:
Its the square of "distance_to"

Reply:
Yeah, that makes sense when you put it like that...

In my head, I was thinking it's the "distance squared"... to a Household. But no, "distance to a Household, squared" makes MUCH more sense and flows way better.

Fixing this now.